cosmoz-grouped-list.js

```diff
@@ -287,16 +313,16 @@
 				templateInstance = element.__tmplInstance;
 			}
 
-			templateInstance.item = item;
+			this._forwardProperty(templateInstance, 'item', item);
 
 			if (isGroup) {
-				templateInstance.selected = this.isGroupSelected(item);
-				templateInstance.folded = this.isFolded(item);
+				this._forwardProperty(templateInstance, 'selected', this.isGroupSelected(item));
+				this._forwardProperty(templateInstance, 'folded', this.isFolded(item));
 			} else {
-				templateInstance.expanded = this.isExpanded(item);
-				templateInstance.selected = this.isItemSelected(item);
+				this._forwardProperty(templateInstance, 'expanded', this.isExpanded(item));
+				this._forwardProperty(templateInstance, 'selected', this.isItemSelected(item));
 			}
-			templateInstance.highlighted = this.isItemHighlighted(item);
+			this._forwardProperty(templateInstance, 'highlighted', this.isItemHighlighted(item), true);
```

Comment:
I think the last call to _forwardProperty with flush = true is confusing.
It might be interpreted as if we flush only this last property.
I would find it more readable if we drop the flush parameter from _forwardProperty, and explicitely flush the properties after the last call. This would also prevent error if ever we add a new fordwarded property in the future.